frontend/src/utilsLogic.tsx

```diff
@@ -384,27 +404,22 @@ export const getTemplatesMapped = (
       persistent,
       resources: { cpu: 0, memory: '', disk: '' },
       instances: instancesSorted || instancesFiltered,
-      workspaceId,
+      workspaceName,
+      workspaceNamespace: 'workspace-' + workspaceName,
```

Comment:
didn't we have workspaceNamespace attribute at a point?

Reply:
i don't believe at this point, here objects have been elaborated many times